sys/dev/drm/drmkpi/drmcompat_dma_buf.c

```diff
@@ -340,6 +340,9 @@ dmabuf_fop_mmap(struct file *file, vm_map_t map, vm_pointer_t *addr,
 	if (foff + size  > dmabuf->size)
 		return (EINVAL);
 
+	if ((*addr + size) >= max_addr)
```

Comment:
Assuming this is the idea of what max_addr is for, this should be > not >=, and you shouldn't have the extra ()